test/typing/plain_files/orm/mapped_column_kwargs.py

```diff
@@ -0,0 +1,90 @@
+from sqlalchemy import Boolean
```

Comment:
this seems very similar to https://github.com/sqlalchemy/sqlalchemy/blob/main/test/typing/plain_files/orm/mapped_column.py as scope, so maybe you can move the same come there?

Reply:
Sure I can merge them